### ISSUE TITLE
Is this alive?

### ISSUE THREAD

**Issue by LaurenceOfAustralia** [tathougies/hos#1]

**Comment by tathougies** [comment]
Yes. However... jhc is no longer actively developed and has several bugs. I've been attempting getting GHC to be more friendly at outputting code that isn't as RTS dependent, but I'm not a GHC-whiz. If you have expertise, I would be open to it.
